easybuild/easyconfigs/n/NGSadmix/NGSadmix-32-GCC-7.3.0-2.30.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'CmdCp'
+
+name = 'NGSadmix'
+version = '32'
+
+homepage = 'http://www.popgen.dk/software/index.php/NgsAdmix'
+description = """This page contains information about the program called 
+NGSadmix, which is a tool for finding admixture proportions 
+from NGS data, based on genotype likelihoods."""
+
+toolchain = {'name': 'GCC', 'version': '7.3.0-2.30'}
+
+# This is a bit of a hack as there is no easyblock to simply download a cpp file and compile it.
+# So we download the file from the source and copy it to the build directory. 
+
+sources = [{
+    'source_urls': ['http://popgen.dk/software/download/NGSadmix/'],
+    'filename': 'ngsadmix32.cpp',
+    'download_filename': 'ngsadmix32.cpp',
```

Comment:
Since filename used for download & saving in the "cache" is the same, there's no need to specify both.

Also, please use `%(version)s` rather than hardcoding `32`, and maybe keep the `source_urls` separate as usual?

```python
source_urls = ['http://popgen.dk/software/download/NGSadmix/']
sources = [{
    'filename': 'ngsadmix%(version)s.cpp',
    'extract_cmd': 'cp %s %(builddir)s'
}]
checksums = ['a5623b71d20114708cfd7ccc5f8c6ce38ff871d1b79af85bc89d38775f2d477e']

Reply:
I changed it as requested. However, I would have thought leaving source_urls is ok:
https://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#common-easyconfig-param-sources-git-config 
Just let me know so I can try and remember it for the next time I am doing something.

Reply:
It would have worked leaving `source_urls` as it was, but using `source_urls` more separately is just more common :)

Reply:
ok. I will try and remember. :-)